src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: External Resources
+description: Configure Okteto's manifest to show external resources in your development environment
+sidebar_label: External Resources
+id: external-resources
+---
+
+import Image from '@theme/Image';
+
+The External Resources feature in Okteto is a powerful mechanism that enables you to manage the lifecycle of cloud external resources as part of your Okteto development environment. This means you can define the external resources that comprise your application, include those resources in Okteto's garbage collection process, and even destroy those resources when you finish working with your development environment.
+
+You no longer need to use different tools to work on your application in Okteto and you won't need to worry about rogue external resources continuing to run once you are done with your work.
+
+When you configure External Resources in Okteto you can see those resources within the Okteto UI, can click to go directly to those resources in their respective platform, and include them in your Okteto `deploy` and `destroy` configurations.
+
+## Overview
+
+When you configure the `external` section in your [Okteto manifest](/reference/manifest), keep in mind the following:
+
+- You can create an image that includes your desired external (to Okteto) resources you'd like to use for your application
+- This image can be included in your Okteto manifest's `deploy` and `destroy` sections
+- External resources are deleted when your Okteto namespace is deleted
+- External resources are covered by Okteto's garbage collection process
+
+## Configuring the `external` section
+
+Let's take a look at the composition of the `external` section in the Okteto manifest starting with the high-level structure of this configuration block. There are some important details you need to know before using this feature.
+
+Here's an example of how to define your `external` resources in the Okteto manifest, which you can see in our [References section](/reference/manifest/#external-object-optional):
+
+```yaml
+external:
```

Comment:
The `external` section is to tell okteto which URL to display. The person defining the resource must also include the necessary steps in `deploy` to provision the resource (e.g. call the AWS API to create an s3 bucket). We don't automate that part today, but we will make samples and building blocks available.

Reply:
Ahhh, that's great clarification. Let me rework this section a bit 👍

Reply:
@rberrelleza Do we automate the `destroy` action or does an external resource also need to be configured there to be destroyed?

Reply:
[Latest commit](https://github.com/okteto/docs/pull/363/files#diff-b5fcc1ba2f90c8eced81c0c31ce915bb44abc500bc7e6f24c7a4a998216e4769R35) attempts to expand on these dynamics.

Let me know if that works 🙏